MEMBER_TRAVEL_FUND.md

```diff
@@ -1,137 +1,101 @@
 ## Purpose
 
-To establish and administer a fund for members of the Node.js
-Foundation to travel and spread knowledge about and support the foundation
-and the Node.js Foundation projects.
+To establish and administer a fund for members of the Node.js Foundation to travel and spread knowledge about and support Node.js ecosystem and the Foundation. 
 
-### [Current Years Fund][]
+### [Current Years Fund](https://github.com/nodejs/admin/blob/master/TravelFunds/2018.md)
```

Comment:
Should this have been updated to 2019.md ? (and if necessary a doc for 2019 created?)

Reply:
will do this under a separate PR